test/fail_compilation/fail4611.d

```diff
@@ -1,16 +0,0 @@
-/*
-TEST_OUTPUT:
----
-fail_compilation/fail4611.d(15): Error: Vec[1000000000] size 4 * 1000000000 exceeds 16MiB size limit for static array
----
-*/
-
-struct Vec
-{
-    int x;
-}
-
-void main()
-{
-    Vec[1000_000_000] a;
-}
```

Comment:
@jacob-carlborg I may need some hand holding. DMD test suite is not something I'm familiar with and trying to figure out what to do from source code is not productive at all (and being undocumented doesn't help it).
Basically I need 'output testing' that will be tested only on Windows.
Any help is appreciated :)